frontend/plugins/accounting_ui/src/modules/transactions/transaction-form/components/common/TBalanceTableRow.tsx

```
@@ -0,0 +1,71 @@
+            >
+              {flexRender(cell.column.columnDef.cell, cell.getContext())}
+            </RecordTable.Cell>
+          ))}
+        </RowComponent>
+      </React.Fragment>
+    )
+  });
+
+  const memoizedTableContent = useMemo(
+    () => tableContent,
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)